### DISCUSSION TITLE
How to only download model without serving it?

### DISCUSSION THREAD

**Discussion by yardenshoham** [vllm-project/vllm#11524] [Q&A]
Currently I run `vllm serve` and look at `~/.cache/huggingface` to get the model. I cancel `vllm serve` once it started. Is there some `vllm pull` command?

**Comment by DarkLight1337** [comment]
I think you can use `huggingface-cli` to download the model directly.

**Comment by kha84** [comment]
`Warning: 'huggingface-cli download' is deprecated. Use 'hf download' instead.`
